src/Microsoft.Azure.WebJobs.Host/Indexers/DefaultBindingProvider.cs

```diff
@@ -25,9 +24,21 @@ internal static class DefaultBindingProvider
             IContextGetter<IMessageEnqueuedWatcher> messageEnqueuedWatcherGetter,
             IContextGetter<IBlobWrittenWatcher> blobWrittenWatcherGetter,
             IExtensionRegistry extensions)
+            
         {
             List<IBindingProvider> innerProviders = new List<IBindingProvider>();
-            innerProviders.Add(new QueueAttributeBindingProvider(nameResolver, storageAccountProvider, messageEnqueuedWatcherGetter));
+#if false
```

Comment:
This is temporary  - it just illustrate where the top-level change is to wire in the new bindings. 
QueueAttributeBindingProvider is the existing provider. 
I'll flip [Queue] over to the new bindings in a separate change